### ISSUE TITLE
The pbench_agent_install.yml failed as the “the role 'pbench_repo_install'” path not right 

### ISSUE THREAD

**Issue by dulili** [distributed-system-analysis/pbench#1791]
The `pbench_agent_install.yml` failed with the below error:
```
ERROR! the role 'pbench_repo_install' was not found in /root/git/pbench/agent/ansible/roles:/root/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/root/git/pbench/agent/ansible
```

The error appears to be in `/root/git/pbench/agent/ansible/pbench_agent_install.yml: line 13, column 7`, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:
```
  roles:
    - pbench_repo_install
      ^ here
```

I checked, the `pbench_repo_install` path changed to `pbench/agent/ansible/pbench/agent/roles/`, it used to be `pbench/agent/ansible/roles/`.

**Comment by ndokos** [comment]
All of the roles have been moved to agree with (what I understand as) Galaxy guidelines, but I neglected to update the instructions on how to install them and how to run the playbooks. I'll forward you an email that I had sent out to a limited audience and I'll update the installation instructions. Sorry for the inconvenience.

**Comment by dulili** [comment]
Got it, Nick
Thanks for your reply!

**Comment by ndokos** [comment]
Here's thje mail that I sent out, included here for convenience. I have not fixed the installation instructions yet. If you try this out, please let me know of any problems you encounter. Thanks!

```
There is a version of the pbench agent roles up on Galaxy finally
(and I have a reproducible, turn-the crank, way to do it again).

Many thanks to Riya for all the help with testing them and finding
problems and above all, figuring out what needs to be done on the
client side, while I was going bananas over 100GB tarballs... Thanks
Riya!

Here are the instructions (an extract from the README in the Github
repo, so don't be concerned about the "above" playbooks not being
there - the whole thing is self contained, but even the extract below
is (mostly) self-contained):

* Installing roles from Ansible Galaxy

The roles that the above playbooks use are available from Ansible
Galaxy, as part of the pbench.agent collection. You can install the
collection with

   ansible-galaxy collection install pbench.agent

The roles will be installed under

    $HOME/.ansible/collections/ansible_collections/pbench/agent

and you will need to add this path to Ansible's roles path in order
for the playbooks to find them:

    ANSIBLE_ROLES_PATH=$HOME/.ansible/collections/ansible_collections/pbench/agent:$ANSIBLE_ROLES_PATH
    export ANSIBLE_ROLES_PATH

or, more permanently, you can add it to the /etc/ansible/ansible.cfg
file.

You will still need a playbook to install the pbench agent - here's the
standard pbench_agent_install.yml from the pbench repo:

---
- name: install pbench-agent
  hosts: servers
  become: yes
  become_user: root

  # The default value ('production') can be overriddent by cenv, a host-specific
  # inventory variable.
  vars:
    pbench_configuration_environment: "{{ cenv | default('production') }}"

  roles:
    - pbench_repo_install
    - pbench_agent_install
    - pbench_agent_config


Once the dust settles a bit, we will probably move the playbooks to Galaxy
as well.

Note that dashes have become underscores throughout: Galaxy did not like dashes
in role names. Please try it out and let me know if there are problems.

```

**Comment by riya-17** [comment]
@ndokos I believe this is resolved?

**Comment by portante** [comment]
@ndokos, as @riya-17 asked earlier, is this issue resolved?

**Comment by ndokos** [comment]
Yes, it is through documentation: the README in agent/ansible mentions the setting of ANSIBLE_ROLES_PATH.

**Comment by portante** [comment]
Fixed by #1757.
